gapps_users_to_contacts_group_copier/gapps_users_to_contacts_group_copier.py

```diff
@@ -4,12 +4,13 @@
 __author__ = "vsin"
 __version__ = "1.0.0"
```

Comment:
By quick Googling, it seems the __author__ string is rarely used, but in all examples it is a string rather than an array (so e.g. `__author__ = "John Doe, Jane Smith, ..."`.) __version__ on the other hand is widely used, and should be probably updated - OTOH this script is not used as a module so it all doesn't really matter much.